### DISCUSSION TITLE
Observing Lithium concentration across positive and negative electrode within a cycle

### DISCUSSION THREAD

**Discussion by mlnsai** [pybamm-team/PyBaMM#2583] [Q&A]
Hi Team,

Is there a way to plot change in Lithium concentration across positive and negative electrode within a cycle (charge discharge cycle) like the way "Electrolyte concentration [mol.m-3]" is being observed?
![1](https://user-images.githubusercontent.com/103929419/209765621-946531f3-258e-4ee7-958c-9a807cd123e5.PNG)

Besides, can you help me understand what exactly positive particle concentration is? Is it same as Lithium concentration in positive electrode?

**Comment by brosaplanella** [comment]
In [this notebook](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/Getting%20Started/Tutorial%203%20-%20Basic%20plotting.ipynb) there are more details on how to plot variables. In this case you want `Positive particle concentration [mol.m-3]`, which will give you the concentration as function of `x`, `r` and `t` (so it will be a colourmap plot). If you want the surface concentration as function of `x` and `t` you can plot `Positive particle surface concentration [mol.m-3]`.
